examples/hlb_cifar10.py

```diff
@@ -160,34 +160,15 @@ def make_square_mask(X, mask_size):
   return mask
 
 def random_crop(X, crop_size=32):
-  mask = make_square_mask(X, crop_size)
+  mask = make_square_mask(X.shape, crop_size)
   mask = mask.repeat((1,3,1,1))
   X_cropped = Tensor(X.flatten().numpy()[mask.flatten().numpy().astype(bool)])
 
   return X_cropped.reshape((-1, 3, crop_size, crop_size))
 
-transform = [
-  lambda x: x.to(device=Device.DEFAULT).float(),
-  lambda x: x / 255.0, # scale
-  lambda x: (x - Tensor(cifar_mean).repeat((1024,1)).T.reshape(1,-1))/ Tensor(cifar_std).repeat((1024,1)).T.reshape(1,-1), # normalize
-  lambda x: x.reshape((-1,3,32,32)),
-  lambda x: pad_reflect(x, size=hyp['net']['pad_amount']),
-  lambda x: random_crop(x, crop_size=32),
-  lambda x: Tensor.where(Tensor.rand(x.shape[0],1,1,1) < 0.5, x[..., ::-1], x), # flip LR
-]
-
-transform_test = [
-  lambda x: x.to(device=Device.DEFAULT).float(),
-  lambda x: x / 255.0,
-  lambda x: (x - Tensor(cifar_mean).repeat((1024,1)).T.reshape(1,-1))/ Tensor(cifar_std).repeat((1024,1)).T.reshape(1,-1),
-  lambda x: x.reshape((-1,3,32,32)),
-]
-
-def cutmix(X, Y, mask_size=3, p=0.5):
-  if Tensor.rand(1) > p: return X, Y
```

Comment:
This for some reason was always true and `cutmix` was never called.

Reply:
thats cuz we don't know what the tensor contents are at this point and `Tensor` overrides the comparison operators to return a `Tensor` which means that this just checked if it was `None`